precommit

```diff
@@ -4,4 +4,6 @@ docker exec tcf_django autopep8 --in-place --aggressive --aggressive -r tcf_webs
 docker exec tcf_django pylint --load-plugins pylint_django --django-settings-module=tcf_core.settings tcf_website tcf_core
 docker exec tcf_django node_modules/.bin/eslint --fix tcf_website/static/**/*.js -c .config/.eslintrc.yml
 # Test.
-docker exec tcf_django python3 manage.py test
+docker exec tcf_django coverage run manage.py test
+# Get code coverage.
+docker exec tcf_django coverage report -m
```

Comment:
I think this is a great addition and could help encourage us to write more tests haha. The only comment I had was should the coverage report be tied with a flag on the ./precommit script? It might be a bit verbose if it's always printed each time.

Reply:
@jackliu612  ["The `-m` flag also shows the line numbers of missing statements."](https://coverage.readthedocs.io/en/v4.5.x/cmd.html) My reasoning was that the `-m` flag will tell you which lines you could test so you will be more tempted to knock if off.